docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,713 @@
+A lot of this variability in the individual estimates comes from short tracks.
+We can show this by including a filtering step (requiring a minimum track length).
+We can filter tracks shorter than a certain length by using :func:`~lumicks.pylake.filter_tracks`::
+
+    min_length = 10
+    methods = {
+        "cve single": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in tracks],
+        "ols single": lambda list_of_tracks: [t.estimate_diffusion("ols") for tracks in list_of_tracks for t in tracks],
+        "cve (filtered)": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in lk.filter_tracks(tracks, min_length)],
+        "ols (filtered)": lambda list_of_tracks: [t.estimate_diffusion("ols") for tracks in list_of_tracks for t in lk.filter_tracks(tracks, min_length)],
+        "cve ensemble": lambda list_of_tracks: [t.ensemble_diffusion("cve") for t in list_of_tracks],
+        "ols ensemble": lambda list_of_tracks: [t.ensemble_diffusion("ols") for t in list_of_tracks],
+        "cve ensemble (filtered)": lambda list_of_tracks: [lk.filter_tracks(t, min_length).ensemble_diffusion("cve") for t in list_of_tracks],
+        "ols ensemble (filtered)": lambda list_of_tracks: [lk.filter_tracks(t, min_length).ensemble_diffusion("ols") for t in list_of_tracks],
+    }
+
+    snrs = 10**np.arange(-1, 1.1, 0.125)
+    results, variances = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+
+    plt.figure(figsize=(12, 4))
+    plt.subplot(1, 2, 1)
+    for method in ("cve single", "ols single", "cve (filtered)", "ols (filtered)"):
+        plot_accuracy(snrs, results[method], variance=variances[method], label=method);
+    plt.ylim([-1, 3])
+
+    plt.subplot(1, 2, 2)
+    for method in ("cve ensemble", "ols ensemble", "cve ensemble (filtered)", "ols ensemble (filtered)"):
+        plot_accuracy(snrs, results[method], variance=variances[method], label=method);
+    plt.ylim([-1, 3])
+
+.. image:: figures/filtering_short_tracks.png
+
+The good news is that the ensemble estimates are pretty robust to filtering short tracks.
+This is expected since these estimates already incorporate weighting (assigning lower weight to short tracks).
+The bad news is that the variance for low SNR is still very high, while the method with the most precision at low SNR (OLS) is very biased.
+Unfortunately, what this means is that we either need to settle for ensemble estimates, or accept high variance results.
+
+Concluding remarks
+------------------
+When the SNR and the number of points per track is high, all methods presented here perform well.
+In this case, the recommendation is to use CVE, while carrying over the localization uncertainty from the ensemble estimate.
+Choosing this option will allow the study of diffusion constants for individual tracks with the largest precision.
```

Comment:
Now imagine a common scenario where a protein is highly diffusive (high SNR) under one condition, and immobile (low SNR) under another condition. What would you advice? It feels awkward to use different methods for the same protein. Maybe using ensemble OLS for both?

Reply:
Actually, since OLS performs better for SNR<1 and all methods perform well for SNR>1, I would tend to advice using OLS?  And only if computation speed is important and SNR>1 shift to CVE? Or am I missing part of the reasoning why CVE is better?

Reply:
I'm also not sure if we can give general advice other than (SNR > 1 go with CVE) to be honest. What I would personally do is probably try all options. If they all agree, the situation is simple. If the results obtained with the methods are different, then I think the best thing to do is to try and simulate your own case with the tools provided in this document and see which method will give you the best answer for your particular data.